widgets/pulseaudio.lua

```diff
@@ -31,7 +31,15 @@ local function factory(args)
     --  '/volume:/ p' prints the line containing 'volume:'.
     --  '/muted:/ p' prints the line containing 'muted:'.
     --  '/device\\.string/ p' prints the line containing 'device.string:'.
-
+    
+    local settings    = args.settings or function()
```

Comment:
But unless most other widgets this won't show _anything_ unless the settings function is defined.